src/pages/register-a-beacon/check-beacon-details.tsx

```diff
@@ -240,42 +153,24 @@ const BeaconHexIdInput: FunctionComponent<FormInputProps> = ({
 
 export const getServerSideProps: GetServerSideProps = withCookieRedirect(
```

Comment:
`getServerSideProps()` ([docs](https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering)) is automatically called by NextJS on every page render.  

It's purpose is to inject dynamic data from the server into the rendered page component.  

Here we're using it to:
- Simultaneously get cached data and update the cache with new POST'd data in `await updateFormCache`.
- If the user just submitted the form, check if it's valid and forward to the next page if so.
- For everything else, send `formData` (cached or otherwise) to the page to be rendered.